org.nicotine_plus.Nicotine.json

```diff
@@ -0,0 +1,103 @@
+{
+  "id" : "org.nicotine_plus.Nicotine",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.36",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "nicotine",
+  "modules" : [
+    "shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json",
+    {
+      "name" : "gsound",
+      "sources" : [
+        {
+          "url" : "https://gitlab.gnome.org/GNOME/gsound.git",
+          "commit" : "a86946b494aa32782b3f6334344d07b5c11d5219",
+          "type" : "git"
+        }
+      ],
+      "cleanup" : [
+        "/bin"
+      ]
+    },
+    {
+      "name" : "gspell",
+      "sources" : [
+        {
+          "url" : "https://download.gnome.org/sources/gspell/1.8/gspell-1.8.3.tar.xz",
+          "sha256" : "5ae514dd0216be069176accf6d0049d6a01cfa6a50df4bc06be85f7080b62de8",
+          "type" : "archive"
+        }
+      ],
+      "cleanup" : [
+        "/bin"
+      ]
+    },
+    {
+      "name" : "python3-miniupnpc",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://files.pythonhosted.org/packages/0c/e8/dbb2747230dfd98a6138cb65b322072eade4d92e1006e518c8711f8f5b85/miniupnpc-2.0.2.tar.gz",
+          "sha256" : "7ea46c93486fe1bdb31f0e0c2d911d224fce70bf5ea120e4295d647dfe274931",
+          "type" : "file"
+        }
+      ],
+      "build-commands" : [
+        "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=\"${FLATPAK_DEST}\" miniupnpc"
+      ]
+    },
+    {
+      "name" : "python3-mutagen",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://files.pythonhosted.org/packages/49/d7/245d48972216d000238d6ccc09142bc1be88e707b38e8fb19a0155292cb6/mutagen-1.44.0-py3-none-any.whl",
+          "sha256" : "1cfc9f40cc0c89f051e3f3dbd5d9057a193c98433cf6c95e02d7f5a395615c01",
+          "type" : "file"
+        }
+      ],
+      "build-commands" : [
+        "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=\"${FLATPAK_DEST}\" mutagen"
+      ],
+      "cleanup" : [
+        "/bin"
+      ]
+    },
+    {
+      "name" : "nicotine",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url" : "https://github.com/Nicotine-Plus/nicotine-plus.git",
+          "commit" : "2e4f3b327d9f68a643d4af1c50570695cd07c1a3",
+          "type" : "git"
+        }
+      ],
+      "build-commands" : [
+        "python3 setup.py install --prefix=\"${FLATPAK_DEST}\""
+      ]
+    }
+  ],
+  "cleanup" : [
+    "/include",
+    "/lib/debug",
+    "/lib/pkgconfig",
+    "/share/aclocal",
+    "/share/info",
+    "/share/man",
+    "/share/gtk-doc",
+    "/share/vala",
+    "*.a",
+    "*.la"
+  ],
+  "finish-args" : [
```

Comment:
We tend to have the finish-args & then cleanup before the modules. It makes review/contributions easier